SecurityPkg/Include/Library/Tpm2CommandLib.h

```diff
@@ -1241,4 +1241,24 @@ Tpm2PcrReadForActiveBank (
   OUT     TPML_DIGEST  *HashList

```

Comment:
Investigate Tianocore version of the different options for the PCR banks. If rotten throw away otherwise upstream